zebra-network/src/peer_set/initialize.rs

```diff
@@ -350,6 +350,9 @@ where
                 }
                 .instrument(handshaker_span),
             );
+
+            // Only one sucesful connection per `MIN_PEER_CONNECTION_INTERVAL` is allowed.
```

Comment:
Expand docs: The inbound peer has already connected to our machine, so all we have control over is the handshake. (And hopefully the OS limits queued inbound connections.)

You were right about the limit only applying to successful connections - and I think that's correct.

```suggestion
            // Only spawn one inbound connection handshake per `MIN_PEER_CONNECTION_INTERVAL`.
            // But clear failed connections as fast as possible.
            //
            // If there is a flood of connections, this stops Zebra overloading the network with handshake data.
            // Most OSes also limit the number of queued inbound connections on a listener port.
```